spot_rl_experiments/configs/config.yaml

```diff
@@ -99,11 +107,14 @@ PLACE_ACTION_SPACE_LENGTH: 4
 SEMANTIC_PLACE_ACTION_SPACE_LENGTH: 9
 SEMANTIC_PLACE_JOINT_BLACKLIST: [3]
 
+# Semantic Place EE
+SEMANTIC_PLACE_EE_ACTION_SPACE_LENGTH: 10
+
 # Open Close Drawer env
 MAX_LIN_DIST_OPEN_CLOSE_DRAWER: 0.0
 MAX_ANG_DIST_OPEN_CLOSE_DRAWER: 3.73 # degrees
 OPEN_CLOSE_DRAWER_ACTION_SPACE_LENGTH: 8
-OPEM_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE: 0.3
+OPEN_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE: 0.3
```

Comment:
did we also make sure that any code that was using `OPEM_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE` (misspelled) uses the new correct spelling `OPEN_CLOSE_DRAWER_DISTANCE_BETWEEN_EE_HANDLE`?

Reply:
Checked it and  resolved